app/static/js/events/scheduler.js

```diff
@@ -859,10 +862,45 @@ function loadMicrolocationsToTimeline(day) {
         window.dayLevelTime.end.minutes = mainEvent.end_time.minutes();
     }
 
-    generateTimeUnits();
-
+    var least_hours = dayLevelTime.start.hours;
+    var least_minutes = dayLevelTime.start.minutes;
+    var max_hours = 0;
+    var max_minutes = 0;
     var dayIndex = _.indexOf(days, day);
 
+    if (/.+\/e\/.+\/schedule\//i.test(document.URL)) {
```

Comment:
This test is so that this changes take place only in the public page and not in the scheduler in event dashboard. Because in event dashboard we need the entire time rather than from start session to end session. Same for the room.